src/Http/Requests/StoreLargoSession.php

```diff
@@ -112,23 +112,18 @@ protected function getAffectedAnnotations($dismissed, $changed)
      */
     protected function imageAnotationsBelongToVolumes($annotations, $volumes)
     {
-        $leni = count($annotations);
-        if ($leni<65000){
-            return !ImageAnnotation::join('images', 'image_annotations.image_id', '=', 'images.id')
-                ->whereIn('image_annotations.id', $annotations)
+        $chunkedI = array_chunk($annotations,config('biigle.db_param_limit')-1); 
```

Comment:
The annotation IDs are not the only parameters in this query. There are also the volume IDs. So the chunk size should be like : `$paramLimit - count($volumes) -  1`.